chia/wallet/cat_wallet/dao_cat_wallet.py

```diff
@@ -159,8 +159,8 @@ async def coin_added(self, coin: Coin, height: uint32, peer: WSChiaConnection, c
         else:
             inner_puzzle = get_innerpuz_from_lockup_puzzle(cat_inner)
             assert isinstance(inner_puzzle, Program)
-            active_votes_list = get_active_votes_from_lockup_puzzle(cat_inner)
-            active_votes_list = [x.as_atom() for x in active_votes_list.as_iter()]
+            active_votes_list_prg = get_active_votes_from_lockup_puzzle(cat_inner)
+            active_votes_list = [bytes32(x.as_atom()) for x in active_votes_list_prg.as_iter()]
```

Comment:
mypy doesn't support reusing variable names with new types. This instance of `active_votes_list` is a `Program` which is different from how it was declared above, as a `List[Optional[bytes32]]`